Detections/AWSCloudTrail/AWS_IngressEgressSecurityGroupChange.yaml

```diff
@@ -21,10 +22,7 @@ query: |
   let timeframe = 1d;
   AWSCloudTrail
   | where TimeGenerated >= ago(timeframe)
-  | where EventName == "AuthorizeSecurityGroupEgress"
-       or EventName == "AuthorizeSecurityGroupIngress"
-       or EventName == "RevokeSecurityGroupEgress"
-       or EventName == "RevokeSecurityGroupIngress"
+  | where EventName in~ ( "AuthorizeSecurityGroupEgress", "AuthorizeSecurityGroupIngress", "RevokeSecurityGroupEgress", "RevokeSecurityGroupIngress")
   | project TimeGenerated, EventName, EventTypeName, UserIdentityAccountId, UserIdentityPrincipalid, UserAgent, 
```

Comment:
Can we summarize here and get an aggregation on certain items to reduce the resulting rowset?

Reply:
Done. Thanks